src/components/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import { NavLink } from 'react-router-dom';
+
+function Nav() {
+  return (
+    <header className="Navbar">
+      <nav className="Nav-header">
+        <h1 className="Nav-logo">
+          <NavLink to="/bookstore/" rel="noopener noreferrer">
+            Bookstore CMS
+          </NavLink>
```

Comment:
## [OPTIONAL]

- [ ] Kindly note that the logo of the website is always linked to the home page. I would suggest you to do the same here. it should link `to="/"`.

Reply:
I have taken note of that, thank you for the suggestion